src/en/pages/index.md

```diff
@@ -1,9 +1,77 @@
 ---
 title: Home
-layout: "layouts/base.njk"
+layout: "layouts/no-aside.njk"
 permalink: /en/
 translationKey: "index"
 redirect_from: /
 ---
 
-# Homepage title
+<header>
+WELCOME
+
+# The Government of Canada Design System
+
+</header>
+
+## Quick Links
+
+<ul>
+    <li>
+        <a href="{{ "/en/components/forms/forms-guidance/" | url }}">Form design</a>
+    </li>
+    <li>
+        <a target="_blank" aria-label="{% if locale == "en" %}Github.com site (Opens in a new tab){% else %}Site Web Github.com (S'ouvre dans un nouvel onglet){% endif %}" href="https://github.com/cds-snc/gcds-components">GitHub repo</a>
```

Comment:
You only need the english version on the english page so the aria-label would be "GitHub repo (Opens in a new tab)"

Reply:
lmao truuuue, it already knows the locale 🤦🏽‍♂️